### PR TITLE
Add pacient

### PR DESCRIPTION
**Pull request by alexandrejuk**

### CODE REVIEW

File: src/controllers/pacient/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const PacientDomain = require('../../domains/pacient')
+
+const pacientDomain = new PacientDomain()
+
+const getPacients = async(req, res, next) => {
+  try {
+    const pacients = await pacientDomain.get()
```
Comment (vitorsilvalima) [inline src/controllers/pacient/index.js]: I think it would be called getAll

File: src/controllers/pacient/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    const pacient = await pacientDomain.getById(req.params.id)
+    res.json(pacient)
+  } catch (error) {
+    next(error)
+  }
+}
+
+const createPacient = async(req, res, next) => {
+  try {
+    const pacientCreated = await pacientDomain.add(req.body)
```
Comment (vitorsilvalima) [inline src/controllers/pacient/index.js]: createdPacient

File: src/controllers/pacient/index.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    res.json(pacientCreated)
+  } catch (error) {
+    next(error)
+  }
+}
+
+const updatePacient = async(req, res, next) => {
+  try {
+    const pacientUpdated = await pacientDomain.updateById(req.params.id, req.body)
```
Comment (vitorsilvalima) [inline src/controllers/pacient/index.js]: updatedPacient

File: src/domains/pacient/index.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  async add(pacient) {
+    return await Pacient.create(pacient)
+  }
+
+  async getById (id){
+    return Pacient.findByPk(id, { raw: true })
+  }
+
+  async get (){
+    return Pacient.findAndCountAll({ raw: true })
```
Comment (vitorsilvalima) [inline src/domains/pacient/index.js]: why use count here?

File: src/controllers/pacient/index.js (hunk 1 above)
Comment (alexandrejuk) [reply src/controllers/pacient/index.js]: I agree

File: src/controllers/pacient/index.js (hunk 3 above)
Comment (alexandrejuk) [reply src/controllers/pacient/index.js]: I agree

File: src/controllers/pacient/index.js (hunk 2 above)
Comment (alexandrejuk) [reply src/controllers/pacient/index.js]: I agree

File: src/domains/pacient/index.js (hunk 4 above)
Comment (alexandrejuk) [reply src/domains/pacient/index.js]: I believe that on future, we have lazy load some table on the front end.

**Comment by vitorsilvalima** [comment]
is it right? the information are different from one to another in each screen
screen shot 2019-01-03 at 16 02 04
screen shot 2019-01-03 at 16 01 56

**Pull request merged**